packages/core/android/src/main/java/io/sentry/react/RNSentryStart.java

```diff
@@ -188,6 +215,42 @@ static void getSentryAndroidOptions(
     setCurrentActivity(currentActivity);
   }
 
+  /**
+   * This function updates the options with RNSentry defaults. These default can be overwritten by
+   * users during manual native initialization.
+   */
+  static void updateWithReactDefaults(@NotNull SentryAndroidOptions options) {
+    // Tracing is only enabled in JS to avoid duplicate navigation spans
+    options.setTracesSampleRate(null);
+    options.setTracesSampler(null);
+    options.setEnableTracing(false);
+  }
+
+  /**
+   * This function updates options with changes RNSentry users should not change and so this is
+   * applied after the configureOptions callback during manual native initialization.
+   */
+  static void updateWithReactFinals(@NotNull SentryAndroidOptions options) {
+    BeforeSendCallback userBeforeSend = options.getBeforeSend();
+    options.setBeforeSend(
+        (event, hint) -> {
+          // Unhandled JS Exception are processed by the SDK on JS layer
+          // To avoid duplicates we drop them in the native SDKs
+          if (event.getExceptions() != null && !event.getExceptions().isEmpty()) {
+            String exType = event.getExceptions().get(0).getType();
+            if (exType != null && exType.contains("Unhandled JS Exception")) {
+              return null; // Skip sending this event
+            }
+          }
```

Comment:
We can omit this JS Exception check because it's set in 

https://github.com/getsentry/sentry-react-native/pull/4451/files#diff-7af7f42daf65903fec96e896fc2518a5422f8545b9bc70c9e512e4da060a589dR199

which should be moved to the RN defaults.

Reply:
Good catch @krystofwoldrich 👍 
Updated with 821fef94291860ba463269804a57016e3a3dec3f